pkg/web/static/js/countrySelect.js

```diff
@@ -1,269 +1,22 @@
-// Display a searchable dropdown for countries.
+import { countriesArray } from './constants.js';
+
+// Display a searchable dropdown for countries in the originator section of the send envelope form.
 const origCountrySelect = new SlimSelect({
   select: '#orig_countries',
   settings: {
     contentLocation: document.getElementById('orig-countries-content'),
   }
 });
 
-// Can more than one country be selected?
+// Display a searchable dropdown for countries in the beneficiary section of the send envelope form.
```

Comment:
I decided to keep these separate for now, but could update the current implementation to use `querySelector`.